Makefile

```diff
@@ -67,8 +67,8 @@ router_setup:
 	$(DOCKER_COMPOSE_CMD) run --rm --no-deps draft-router-api bundle exec rake db:purge
 
 content_store_setup:
-	$(DOCKER_COMPOSE_CMD) run --rm --no-deps content-store bundle exec rake db:purge
-	$(DOCKER_COMPOSE_CMD) run --rm --no-deps draft-content-store bundle exec rake db:purge
+	$(DOCKER_COMPOSE_CMD) run --rm --no-deps content-store bundle exec rake db:setup
```

Comment:
Do we still need purge (or just reset) to maintain the same behaviour?

Reply:
Oh yes, that's a good point.

Reply:
Updated